### ISSUE TITLE
[Keyboard Navigation - Logic apps - Design]: Incorrect role is defined as image for controls under "Design" section.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6099]
### Describe the issue 

User Experience: 

 Visually impaired users who depend on screen reader will find difficulty in understanding the functionality of the control if correct role will not be defined for them, as a result they will be misguided and might get blocked from further actions.

 Note: User credentials should NOT be included in the bug.

Pre-requisites:

- Make sure dependencies such as Azure Logic Apps (Standard), Azure Functions, C# are installed. 
- Navigate to Azure control present on the right side and select it. 
- Navigate to LogicApp icon present under workspace section, click on tat and select 'Create New Project'. 
- Navigate select stateful workflow, then enter the name of the workflow. 
- Once the workflow has been created then enter the name of the created workflow in edit field present on the top and press Enter. 
- Navigate to 'Open in current window' and select it. 
- A dialog will appear, select 'Yes', now the workflow will open in new window. 

Repro Steps:

- Open Visual Studio app. 
- Navigate to any workflow and right click on it, then select 'open designer'. 
- Navigate all the controls of the designer screen. 
- Observe if Incorrect role is defined as image for controls under "Design" section. 

Actual Result: 

 Incorrect role is defined as image for controls such as "Secure inputs or outputs enabled" and "Connection required" present under "Design" section. Screen reader announces as "Graphic" but on selecting with them enter key new dialog gets activated.

Expected result: 

 Correct role should be defined for controls such as "Secure inputs or outputs enabled" and "Connection required" present under "Design" section as "button".

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
https://msazure.visualstudio.com/One/_workitems/edit/29367611
